pyvo/dal/scs.py

```diff
@@ -426,10 +426,6 @@ class SCSResults(DALResults, DatalinkResultsMixin):
     :py:class:`~pyvo.dal.scs.SCSRecord` instances) are typically
     accessed by iterating over an ``SCSResults`` instance.
 
-    >>> results = pyvo.conesearch(url, pos=[12.24, -13.1], radius=0.1)
-    >>> for src in results:
-    ...     print("{0}: {1} {2}".format(src.id, src.ra, src.dec))
-
     Alternatively, records can be accessed randomly via
```

Comment:
No place for these here?

Reply:
It was inconsistent, some classes had them, some didn't. And they are not runnable as is, but would need to be fully extended, and the class itself is not mentioned in the example. So I went for the easy way and removed them all. 
With long, and narrative docstrings like this, I would rather suggest pointing the users towards the narrative docs?